From ce339cd23ec9ebf24482e68bd784dc443217385e Mon Sep 17 00:00:00 2001
From: Leonardo Taccari <iamleot@gmail.com>
Date: Thu, 31 Oct 2024 14:13:45 +0100
Subject: [PATCH] Fix windows_to_olson inclusion

In icalendar-6.0.0 windows_to_olson is part of icalendar.timezone.
---
 khal/icalendar.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/khal/icalendar.py b/khal/icalendar.py
index 3d1fcd71f..a50e5f691 100644
--- a/khal/icalendar.py
+++ b/khal/icalendar.py
@@ -59,8 +59,8 @@ def split_ics(ics: str, random_uid: bool=False, default_timezone=None) -> List:
         # Standard Time' for 'Pacific/Auckland' in Olson format), we convert any
         # Windows format timezones to Olson.
         if item.name == 'VTIMEZONE':
-            if item['TZID'] in icalendar.windows_to_olson.WINDOWS_TO_OLSON:
-                key = icalendar.windows_to_olson.WINDOWS_TO_OLSON[item['TZID']]
+            if item['TZID'] in icalendar.timezone.windows_to_olson.WINDOWS_TO_OLSON:
+                key = icalendar.timezone.windows_to_olson.WINDOWS_TO_OLSON[item['TZID']]
             else:
                 key = item['TZID']
             tzs[key] = item
